.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,15 @@
-## Description
-<!-- Add a description of your PR here.
-A good description should include pointers to an issue or design document, etc.
--->
-## Upgrade Notes
-Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
-* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)
-
-Does this PR fix a zero-downtime upgrade introduced earlier?
-* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)
-
-Does this PR otherwise need attention when creating release notes? Things to consider:
+Instructions:
+1. The PR has to be tagged with at least one of the following labels:
+   1. `feature`
+   2. `bugfix`
+   3. `performance`
+   4. `ui`
+   5. `backward-incompat`
+   6. `release-notes` (*)
```

Comment:
Add labels:
Testing
Dependencies 
Deployment 
Observability
Security 
Code style

Reply:
I would say add a lot of text in "Contribution guidelines" section in the docs, and put a link to that here? 

We decide, of course :)

Reply:
I agree, the actual problem being solved here is PRs being merged without labeling. It isn't a contributor problem. So I would argue against having a template at all (the list of labels in the template would either be incomplete or very long), and instead document how PRs should be labeled in the contribution guidelines.

Reply:
ca we add atleast some of the labels suggested by Richard? I still see only the initial 6. At least testing, refactor, cleanup, else many PRs would simply get tagged as feature/bug-fix even when they're not